controllers/chat/sendAnonymousMessage.js

```diff
@@ -134,19 +134,18 @@ const sendAnonymousMessage = async (req, res) => {
       dataPayload.is_big_message = 'true';
       dataPayload.attachment = '';
     }
-    channelMember.forEach((member) => {
-      let payload = {
+
+    BetterSocialCore.fcmToken.sendChatNotificationByChannelMembers(
+      channelMember,
+      {
         notification: notificationPayload,
-        data: {
-          ...dataPayload,
-          is_annoymous: member.is_anonymous.toString()
-        }
-      };
-      BetterSocialCore.fcmToken.sendChatNotification(member.user_id, payload, {
+        data: dataPayload
+      },
+      {
         priority: 'high',
         content_available: true
-      });
-    });
+      }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new implementation of sending notifications to channel members seems to be more efficient as it sends notifications in a batch rather than individually. However, the `is_anonymous` field is missing in the new data payload. This field was present in the old code and might be important for handling anonymous messages. Please ensure that removing this field won't cause any unintended side effects.
```diff
    BetterSocialCore.fcmToken.sendChatNotificationByChannelMembers(
      channelMember,
      {
         notification: notificationPayload,
        data: {
+               ...dataPayload,
+               is_annoymous: member.is_anonymous.toString()
        }
      },
      {
         priority: 'high',
         content_available: true
      }
    );
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->